app/views/spouse_auth_only/show.html.erb

```diff
@@ -26,7 +26,7 @@
       If you have any trouble please e-mail <a href="mailto:hello@getyourrefund.org">hello@getyourrefund.org</a>. We’re here to help!
     </p>
 
-    <%= link_to user_idme_omniauth_authorize_path(spouse: "true"), class: "button button--primary button--wide text--centered", method: :post do %>
+    <%= link_to destroy_idme_session_path, class: "button button--primary button--wide text--centered", method: :delete do %>
```

Comment:
maybe we should leave a comment just to clarify why we leave a sign out link on this page? Might be surprising to future developers.

Reply:
done!